lib/travis/api/v3/renderer/subscription.rb

```diff
@@ -10,4 +10,13 @@ class Renderer::BillingInfo < ModelRenderer
   class Renderer::CreditCardInfo < ModelRenderer
     representation(:minimal, :card_owner, :expiration_date, :last_digits)
   end
+
+  class Renderer::Invoices < CollectionRenderer
+    type  :invoices
+    collection_key :invoices
+  end
+
+  class Renderer::Invoice < ModelRenderer
+    representation(:standard, :id, :created_at, :url)
+  end
```

Comment:
We added `BillingInfo` and `CreditCardInfo` renderers in this file because they are only used embedded on the `Subscription` (in fact we tried to have them inside the `Subscription` namespace but that weirds the class discovery mechanism out).

Since `Invoice` is a standalone renderer, I'd suggest it goes in its own file(s).